dlc-wallet/src/main/resources/postgresql/dlc/migration/V1__dlc_db_baseline.sql

```diff
@@ -1,33 +1,33 @@
 CREATE TABLE "global_dlc_data"
 (
-    "dlc_id"              TEXT PRIMARY KEY,
-    "temp_contract_id"    TEXT    NOT NULL UNIQUE,
-    "contract_id"         TEXT UNIQUE,
-    "protocol_version"    INTEGER NOT NULL,
-    "state"               TEXT    NOT NULL,
-    "is_initiator"        INTEGER NOT NULL,
-    "account"             TEXT    NOT NULL,
-    "change_index"        INTEGER NOT NULL,
-    "key_index"           INTEGER NOT NULL,
+    "dlc_id"                TEXT PRIMARY KEY,
+    "temp_contract_id"      TEXT    NOT NULL UNIQUE,
+    "contract_id"           TEXT UNIQUE,
+    "protocol_version"      INTEGER NOT NULL,
+    "state"                 TEXT    NOT NULL,
+    "is_initiator"          INTEGER NOT NULL,
+    "account"               TEXT    NOT NULL,
+    "change_index"          INTEGER NOT NULL,
+    "key_index"             INTEGER NOT NULL,
 
-    "fee_rate"              VARCHAR(254) NOT NULL,
-    "fund_output_serial_id" INTEGER      NOT NULL,
+    "fee_rate"              TEXT    NOT NULL,
+    "fund_output_serial_id" TEXT    NOT NULL,
```

Comment:
I don't think we are able to write a valid v2 migration, idk if we can get the proper uint64 hex encoding from sql's integer, and even if we can, will require decontruscting and then rebuilding the entire db because the `global_dlc_data` has a serial id in it